EPG fetch from NextPVR crashes Jellyfin

The nightly EPG fetch done by jellyfin using the NextPVR results in an exception in Jellyfin followed by a restart. The result is that you only see a small part of the EPG. This started relatively recently after a Jellyfin version upgrade.

See attached Jellyfin log.[jellyfin20241223.log](https://github.com/user-attachments/files/18226622/jellyfin20241223.log)

@emveepee sorry I already replied in my edit of https://github.com/jellyfin/jellyfin-plugin-nextpvr/issues/59#issuecomment-2646617566. TLDR; I only checked LiveTvService.cs and all other exception handler catch "Exception" so they handle all exceptions.
So I did a quick check and other files seems fine:
```
git grep catch
.editorconfig:csharp_new_line_before_catch = true
Jellyfin.Plugin.NextPVR/LiveTvService.cs:                catch (Exception ex)
Jellyfin.Plugin.NextPVR/LiveTvService.cs:                catch (Exception ex)
Jellyfin.Plugin.NextPVR/LiveTvService.cs:        catch (HttpRequestException)
Jellyfin.Plugin.NextPVR/LiveTvService.cs:        catch (TaskCanceledException)
Jellyfin.Plugin.NextPVR/RecordingsChannel.cs:            catch (Exception)
Jellyfin.Plugin.NextPVR/RecordingsChannel.cs:                catch (Exception)
Jellyfin.Plugin.NextPVR/Responses/InstantiateResponse.cs:        catch
Jellyfin.Plugin.NextPVR/Responses/RecordingResponse.cs:        catch (Exception err)
Jellyfin.Plugin.NextPVR/Responses/RecordingResponse.cs:        catch (Exception err)
jellyfin.ruleset:        <!-- disable warning CA1031: Do not catch general exception types -->
```
LiveTvService is the only one not to handle all exceptions at once.
Maybe I should do the same for GetLastUpdate ie replace HttpRequestException by Exception instead of handling both HttpRequestException and TaskCanceledException).

Note that I have not been able to test my PR yet.

--- a/Jellyfin.Plugin.NextPVR/LiveTvService.cs
+++ b/Jellyfin.Plugin.NextPVR/LiveTvService.cs
@@ -621,10 +621,10 @@ public class LiveTvService : ILiveTvService
 
             UtilsHelper.DebugInformation(_logger, $"GetLastUpdateTime {retTime.ToUnixTimeSeconds()}");
         }
-        catch (HttpRequestException)
+        catch (Exception ex)
         {
             LastUpdatedSidDateTime = DateTimeOffset.MinValue;
-            _logger.LogWarning("Could not connect to servier");
+            _logger.LogWarning(ex, "Could not connect to server");
             Sid = null;
         }
 

